index.css

```diff
@@ -385,6 +348,150 @@ li {
   margin-top: 1em;
 }
 
+.modal {
+  display: none;
+  position: fixed;
+  border-radius: 20px;
+  z-index: 10;
+  top: 0;
+  width: 90%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+  left: 0;
+  margin: 0;
+}
+
+.modal-buttons {
+  display: flex;
+  justify-content: center;
+}
+
+.close-modal {
+  display: inline !important;
+  color: var(--title-color);
+  font-size: x-large;
+  margin: 0;
+}
+
+.modal-title-close h2 {
+  width: fit-content;
+  margin: 0;
+  display: inline-block;
+}
+
+.modal-title-close {
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  height: 50px;
+}
+
+.modal-top {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 1em;
+}
+
+@media (min-width: 375px) {
+  .modal {
+    display: none;
+    position: fixed;
+    border-radius: 20px;
+    z-index: 10;
+    top: 0;
+    width: 100vw;
+    height: 100vh;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+    left: 0;
+    margin: 0;
+    padding: 0;
+    animation: popupSlideDown 0.3s;
+  }
```

Comment:
- [OPTIONAL] You've done well in incorporating the revisions recommended by the previous reviewer :+1: , but I believe it would be more appropriate to position the popup at the center of your webpage so that it aligns with the Figma template. This means that the popup should be placed in the middle of the screen, both horizontally and vertically, so that it is more aesthetically pleasing and consistent with the original design. This will help ensure that the webpage looks cohesive and professional, while also enhancing the overall user experience.
- [OPTIONAL] It seems like there are some issues with the images in your portfolio, as they are not being displayed properly

https://user-images.githubusercontent.com/105117832/224019142-6f6712bd-9f3e-40ef-a0c3-d4b4f6b486e3.mp4